modules/media/help/media.md

```diff
@@ -1,43 +1,17 @@
 # Media
 
-The Media module serves as a repository for files associated with a particular 
-candidate-timepoint or instrument in a study.  Users can access and upload files 
-such as PDF scans, recordings, log files, or stimulus presentation media in multiple 
-formats such as .pdf, .mp4, .mp3, .txt, etc. 
-
-## Searching for a Media File
-
-Under the Browse tab, use the Selection Filters to search for files by fields such 
-as filename, file type, candidate PSCID, Visit Label, Instrument, Site, or 
-name of user who uploaded the file.  Partial string matching can be used on many 
-fields (e.g. filename).  As filters are selected, the data table below will
-dynamically update with relevant results.  Click the “Clear Filters” button to reset 
-all filters. 
-
-Within the data table, results can be sorted in ascending or descending order by 
-clicking on any column header. To download a file, click on the filename (in blue 
-text). 
-
-## Uploading a Media File
-
-Under the “Upload” tab, users will be able to upload a new media file. 
-Users must specify information about the file by selecting from a number of dropdown 
-menus including PSCID, Visit Label, Site, Instrument (optional), Date of 
-Administration, and enter comments on the file in the textbox provided. Lastly, 
-users will select the file they wish to upload by clicking the “Browse” button and 
-selecting the file from their computer. To begin the upload, users must click 
-“Upload File”. 
-
-Note that a file to be uploaded should follow the naming convention listed at the 
-top of the Upload page: [PSCID]\_[VisitLabel]\_[Instrument]
-
-## Editing a Media File
-
-Click “Edit” from the “Edit Metadata” column to update or edit certain file 
-properties, via the “Edit Media File” page. 
-
-This page allows users to modify information about an uploaded file.  Fields such 
-as Date of Administration, Comments, and file visibility (hidden or visible) can be 
-updated.  Note that fields such as PSCID, Visit Label, Site, and Instrument cannot 
-be modified on uploaded media files. To finalize an update, users must click “Update 
-File”. 
+This module serves as a repository for files associated with a particular candidate-timepoint or instrument in a study. You can access and upload files such as PDF scans, recordings, log files, or stimulus presentation media in a variety of formats formats, including .pdf, .mp4, .mp3, .txt, and more.
+
+## Search For/Edit a File
```

Comment:
The `Search for...` sub-header here and other sub-headers are bigger than the `Media` module name  -- taking a look at other modules, this looks like an inconsistency here